problem_builder/templates/html/tip_choice_group.html

```diff
@@ -3,4 +3,4 @@
     {{ tip_html|safe }}
   {% endfor %}
 </div>
-<button class="close icon-remove-sign fa fa-times-circle"></button>
+<button class="close icon-remove-sign fa fa-times-circle">&times;</button>
```

Comment:
Shouldn't we use `icon-remove-sign fa fa-times-circle` **or** `&times;` to avoid workarounds to these duplicated buttons?

Reply:
Removing a class may break some css rules if any other party is also using this xblock. So I wanted to keep the things intact to avoid any issue

Reply:
Right, good point :+1: